extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -313,23 +313,23 @@
 # to complex queries. The following settings affect all queries, wherever they
 # occur.
 ##
-$GLOBALS['smwgQEnabled'] = true;   // (De)activates all query related features and interfaces
-$GLOBALS['smwgQMaxLimit'] = 10000; // Max number of results *ever* retrieved, even when using special query pages.
-$GLOBALS['smwgIgnoreQueryErrors'] = true; // Should queries be executed even if some errors were detected?
+$smwDefaults['smwgQEnabled'] = true;   // (De)activates all query related features and interfaces
+$smwDefaults['smwgQMaxLimit'] = 10000; // Max number of results *ever* retrieved, even when using special query pages.
+$smwDefaults['smwgIgnoreQueryErrors'] = true; // Should queries be executed even if some errors were detected?
 										// A hint that points out errors is shown in any case.
 
-$GLOBALS['smwgQSubcategoryDepth'] = 10;  // Restrict level of sub-category inclusion (steps within category hierarchy)
-$GLOBALS['smwgQSubpropertyDepth'] = 10;  // Restrict level of sub-property inclusion (steps within property hierarchy)
+$smwDefaults['smwgQSubcategoryDepth'] = 10;  // Restrict level of sub-category inclusion (steps within category hierarchy)
+$smwDefaults['smwgQSubpropertyDepth'] = 10;  // Restrict level of sub-property inclusion (steps within property hierarchy)
 										// (Use 0 to disable hierarchy-inferencing in queries)
-$GLOBALS['smwgQEqualitySupport'] = SMW_EQ_SOME; // Evaluate #redirects as equality between page names, with possible
+$smwDefaults['smwgQEqualitySupport'] = SMW_EQ_SOME; // Evaluate #redirects as equality between page names, with possible
 												// performance-relevant restrictions depending on the storage engine
-// $GLOBALS['smwgQEqualitySupport'] = SMW_EQ_FULL; // Evaluate #redirects as equality between page names in all cases
-// $GLOBALS['smwgQEqualitySupport'] = SMW_EQ_NONE; // Never evaluate #redirects as equality between page names
-$GLOBALS['smwgQSortingSupport']     = true; // (De)activate sorting of results.
-$GLOBALS['smwgQRandSortingSupport'] = true; // (De)activate random sorting of results.
-$GLOBALS['smwgQDefaultNamespaces'] = null; // Which namespaces should be searched by default?
+// $smwDefaults['smwgQEqualitySupport'] = SMW_EQ_FULL; // Evaluate #redirects as equality between page names in all cases
+// $smwDefaults['smwgQEqualitySupport'] = SMW_EQ_NONE; // Never evaluate #redirects as equality between page names
+$smwDefaults['smwgQSortingSupport']     = true; // (De)activate sorting of results.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)